src/EnergyPlus/UnitarySystem.cc

```diff
@@ -11221,8 +11247,9 @@ namespace UnitarySystems {
         DataHVACGlobals::CompressorOperation CoolingCompOn = DataHVACGlobals::CompressorOperation::Off;
         if (CoolPLR > 0) {
             CoolingCompOn = CompressorOn;
-            // for multispeed coils, comp is on IF speed > 1
-        } else if (this->m_CoolingSpeedNum > 1) {
+        } else if (CoolPLR == 0 && this->m_EconoSpeedNum > 0) { // turn compressor off for economizer calculations
+            CoolingCompOn = DataHVACGlobals::CompressorOperation::Off;
+        } else if (this->m_CoolingSpeedNum > 1) { // for multispeed coils, comp is on IF speed > 1
             CoolingCompOn = DataHVACGlobals::CompressorOperation::On;
         }
```

Comment:
I think that's trick how you inserted the economizer control into these IF blocks. It's just hard to know how the logic responds to both choices of econo control.

    if (CoolPLR > 0) {                 <- compressor is on
    } else if (CoolPLR == 0 && this->m_EconoSpeedNum > 0) { // turn compressor off for economizer calcs
    } else if (this->m_CoolingSpeedNum > 1) { // for multispeed coils, comp is on IF speed > 1

Like I said on the conference call, these new concepts are hard to nail down initially and it takes a few release cycles to figure out what we did wrong. And we don't know that until users come back and say "Hey, what the heck is going on here?"

From this inspection, this one looks OK (famous last words).